src/Components/CreateImageWizardV2/steps/Review/targetEnvironment.tsx

```diff
@@ -0,0 +1,258 @@
+      <br />
+    </TextContent>
+  );
+};
+
+type TargetEnvGCPListPropTypes = {
+  accountType: string;
+  accountEmail: string;
+  domain: string;
+};
+
+export const TargetEnvGCPList = ({
+  accountType,
+  accountEmail,
+  domain,
+}: TargetEnvGCPListPropTypes) => {
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>GCP</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Account type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {accountType}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          {accountType === 'Domain' ? 'Domain' : 'Principal'}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {accountEmail || domain}
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+type TargetEnvAzureListPropTypes = {
+  manual: boolean;
+  sourceId: number;
+  tenantId: string;
+  subscriptionId: string;
+  resourceGroup: string;
+};
+
+export const TargetEnvAzureList = ({
+  manual,
+  sourceId,
+  tenantId,
+  subscriptionId,
+  resourceGroup,
+}: TargetEnvAzureListPropTypes) => {
+  const { data, isSuccess } = useGetSourceListQuery({
+    provider: 'azure',
+  });
+  const sources = data !== undefined ? extractProvisioningList(data) : [];
+  const [sourceName, setSourceName] = useState(
+    getSourceName(sources, sourceId.toString())
+  );
+  useEffect(() => {
+    if (!sources) {
+      return;
+    }
+    const val = getSourceName(sources, sourceId.toString());
+    if (val) {
+      setSourceName(val);
+    }
+  }, [sources]);
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>Microsoft Azure</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        {!manual && isSuccess && (
+          <>
+            <TextListItem component={TextListItemVariants.dt}>
+              Azure Source
+            </TextListItem>
+            <TextListItem component={TextListItemVariants.dd}>
+              {sourceName}
+            </TextListItem>
+          </>
+        )}
+        {manual && (
+          <>
+            <TextListItem component={TextListItemVariants.dt}>
+              Azure Tenant ID
```

Comment:
Maybe GUID again? So we keep the labels the same.

```suggestion
              Azure Tenant GUID
```